app/controllers/questions_controller.rb

```diff
@@ -19,9 +23,111 @@ def show
       gon.solution = solution
     end
 
+    if !(current_user.show_solutions)
+      if @room.answered(@question.id,@room.id)
+        @number_of_question = @room.question_count - @room.question_count_not_visited(@room.id) + 1
+      else
+        @number_of_question = @room.question_count - @room.question_count_not_visited(@room.id)
+      end
+    else
+      @number_of_question = @room.learning_objects.where("learning_object_id < ?",@question.id).count + 1
```

Comment:
Otazky v miestnosti sa usporaduvaju podla IDcka? (Ked sa cislo otazky robi cez where a mensie id.)
To nemusi byt najstastnejsie riesenie, kedze potom pojdu podla poradia v akom vznikali v ramci jednotlivych skupin (vid CSVcka) a zaroven pojdu bud najskor vsetky ne-evaluator a potom evaluator alebo naopak.